deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -1160,7 +1160,7 @@ spec:
                   - hyperconverged-cluster-operator
                   env:
                   - name: OPERATOR_IMAGE
-                    value: rthallisey/hyperconverged-cluster-operator:latest
+                    value: docker.io/hyperconverged-cluster-operator:latest
```

Comment:
Something isn't right here with this image value.

Reply:
its right with not using @rthallisey private one ;-)